tests/features/site-switcher-settings.feature

```diff
@@ -0,0 +1,33 @@
+@api
+Feature: Site switcher settings
+  In order to be able to set which site switcher link should be active
+  As a privileged user
+  I want to be able to access the site switcher settings page and update its values
+
+  @preserve-site-switcher-configuration
```

Comment:
Won't need tag since you use the config context

Reply:
I turns our we still need it since we are changing configuration via UI.